forte/data/readers/base_reader.py

```diff
@@ -50,15 +50,15 @@ class BaseReader(PipelineComponent[PackType], ABC):
 
     def __init__(self,
                  from_cache: bool = False,
-                 cache_directory: Optional[Path] = None,
+                 cache_directory: Optional[str] = None,
                  append_to_cache: bool = False):
         """
         Args:
             from_cache (bool, optional): Decide whether to read from cache
-                if cache file exists. By default (``True``), the reader will
-                try to read an datapack from the first line of the caching file.
-                If ``False``, the reader will only read from the original file
-                and use the cache file path only for output.
+                if cache file exists. By default (``False``), the reader will
+                only read from the original file and use the cache file path
+                only for output. If ``True``, the reader will try to read a
+                datapack from the first line of the caching file.
```

Comment:
Actually I don't understand "the first line of the caching file", I think that is not correct.

Reply:
it's what the current documentation says..I've just rearranged it. will fix it.